.github/renovate.json

```diff
@@ -10,7 +10,8 @@
   "ignorePresets": [
     ":dependencyDashboard",
     ":ignoreModulesAndTests",
-    ":semanticPrefixFixDepsChoreOthers"
+    ":semanticPrefixFixDepsChoreOthers",
+    "docker:pinDigests"
```

Comment:
Docker images are used only for tests, I don't think we want to pin their digests.